nixos/tests/conduit.nix

```
@@ -0,0 +1,27 @@
+import ./make-test-python.nix ({ pkgs, ... } : let
+
+
+  name = "conduit";
+
+in {
+
+
+  nodes = {
+    serverconduit = args: {
```

```suggestion
    conduit = args: {
```

This should be clear enough, no?